### ISSUE TITLE
Claiming namespace omergulen

### ISSUE THREAD

**Issue by omergulen** [EclipseFdn/open-vsx.org#1924]
`omergulen` is my name and surname, I already published my extension with this publisher name on [VS Marketplace](https://marketplace.visualstudio.com), and due to requests from users, I want to publish them on Open-VSX as well.

**Comment by kineticsquid** [comment]
@omergulen Thanks for your contribution!

**Comment by omergulen** [comment]
Thank you @kineticsquid!

**Comment by omergulen** [comment]
Just a quick question, why do you think I still have the warning on top of my extension that was published under the `omergulen` space?

https://open-vsx.org/extension/omergulen/universal-console-utils

Is the extension page is statically generated, do we need to deploy recent build?

**Comment by kineticsquid** [comment]
@omergulen No, the pages are dynamic, though there are caches involved and sometimes the changes take a little while to show up. Are you still seeing the warning? This is what I see:
<img width="1325" alt="image" src="https://github.com/EclipseFdn/open-vsx.org/assets/681707/ad748da4-1476-4642-8005-c75ee8723716">

**Comment by omergulen** [comment]
I just checked it and the issue seems to be gone, thank you for the explanation and checking, have a good one!
